src/System.ServiceModel.Primitives/ref/project.json

```diff
@@ -13,6 +13,8 @@
     "System.Reflection": "4.0.0",
     "System.Net.Primitives": "4.0.0",
     "System.Security.Principal": "4.0.0",
+    "System.Security.Cryptography.X509Certificates" : "4.0.0-beta-*",
```

Comment:
Unfortunately, we don't have a 4.0.0 version of this contract published, per http://www.nuget.org/packages/System.Security.Cryptography.X509Certificates. 

So yes, we will probably need to bump the patch version (i think... we shouldn't need to bump the minor) of this contract. Any concerns with that? Otherwise we'd have to wait until the 4.0.0 stable release comes out, whenever that is, before a dependency can be taken.